### ISSUE TITLE
[FEATURE] Dense account design

### ISSUE THREAD

**Issue by Jack-Works** [Ivy-Apps/ivy-wallet#3297]
### Please confirm the following:

- [X] I've checked [the current issues](https://github.com/Ivy-Apps/ivy-wallet/issues) for duplicate issues.
- [X] I've requested a single (only one) feature/change in this issue. It complies with the One Request Per GitHub Issue (ORPGI) rule.
- [X] My issue is well-defined and describes how it should be implemented from UI/UX perspective.

### What do you want to be added or improved?

<img width="370" alt="image" src="https://github.com/Ivy-Apps/ivy-wallet/assets/5390719/27c8fd22-8ff1-4fcd-93a6-9e63bfd203ef">

It's inefficient to show only 3 accounts on a screen. It will be good if it can show more.

**Comment by ILIYANGERMANOV** [comment]
Solution: Remove "INCOME THIS MONTH" and "EXPENSES THIS MONTH".

**Comment by ivywallet** [comment]
Thank you @Jack-Works for raising Issue #3297! 🚀
What's next? Read our **[Contribution Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/CONTRIBUTING.md) 📚**.

_Tagging @ILIYANGERMANOV for review & approval 👀_

**Comment by Jack-Works** [comment]
> Solution: Remove "INCOME THIS MONTH" and "EXPENSES THIS MONTH".

I can imagine a more dense design that can preserve those.

<img width="377" alt="image" src="https://github.com/Ivy-Apps/ivy-wallet/assets/5390719/7dbc022a-9bfd-4d36-a5f0-08a028235b0e">

**Comment by ILIYANGERMANOV** [comment]
> > Solution: Remove "INCOME THIS MONTH" and "EXPENSES THIS MONTH".
> 
> I can imagine a more dense design that can preserve those.
> 
> <img alt="image" width="377" src="https://private-user-images.githubusercontent.com/5390719/346353126-7dbc022a-9bfd-4d36-a5f0-08a028235b0e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjAzNzc4MjcsIm5iZiI6MTcyMDM3NzUyNywicGF0aCI6Ii81MzkwNzE5LzM0NjM1MzEyNi03ZGJjMDIyYS05YmZkLTRkMzYtYTVmMC0wOGEwMjgyMzViMGUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDcwNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA3MDdUMTgzODQ3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NmVmOThkMjk4Y2FiZThiZjAwZDc4ODczMDU1NTU1ZGY5ZDYwYWNkMjY1NDBlNzJhNTBmY2NhYzZhMjNhZTQwNiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.dHL6w5z2DVN7zGKk4rIL7hntuXCYGU9k0aRr-LxWrEw">

We still need to drop the income/expense numbers because:
- they don't reflect the selected period on the home screen
- we don't see their currencies
- it's too much information for that screen

@stefanilijev97 can come up with an elegant design solution for this but in general we need a more compact design for sure

**Comment by vraj152** [comment]
I was going to open the similar issue, but then found this! 

I think same applies to "Categories" view too. Most if not all, categories will have "expense" (not income) 

Consider this screenshot. The space here is mostly wasted. 

![Screenshot_20240722-231543](https://github.com/user-attachments/assets/8253eedc-fe78-4f0b-a772-230bb61b35c3)

**Comment by ILIYANGERMANOV** [comment]
@vraj152 please open a new issue so different contributors can work in parallel. Otherwise, one contributor will fix the accounts, close the issue and we'll forget about the categories

**Comment by shamim-emon** [comment]
I'm on it

**Comment by ivywallet** [comment]
Thank you for your interest @shamim-emon! 🎉
Issue #3297 is assigned to you. You can work on it! ✅

_If you don't want to work on it now, please un-assign yourself so other contributors can take it._

Also, make sure to read our [Contribution Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/CONTRIBUTING.md).
